unittests/producer_schedule_if_tests.cpp

```diff
@@ -20,7 +20,7 @@ inline account_name get_expected_producer(const vector<producer_authority>& sche
 
 } // anonymous namespace
 
-BOOST_FIXTURE_TEST_CASE( verify_producer_schedule_after_instant_finality_activation, validating_tester ) try {
+BOOST_FIXTURE_TEST_CASE( verify_producer_schedule_after_instant_finality_activation, legacy_validating_tester ) try {
```

Comment:
Maybe add a comment that this uses `legacy_validating_tester` because it transitions to savanna as part of the test.